src/webui/src/components/overview/command/Command2.tsx

```diff
@@ -11,7 +11,10 @@ export const Command2 = (): any => {
     if (clusterMetaData !== undefined) {
         for (const item of clusterMetaData) {
             if (item.key === 'command') {
-                trialCommand = item.value;
+                trialCommand = item.value as string;
+            }
+            if (item.key === 'trial_config') {
+                trialCommand = item.value['command'];
```

Comment:
if 'command' field does not exist in trial_config, will this line cause webui crash?

Reply:
Why not `item.value.command`?

Reply:
had fix this question